modifier/collection/move.go

```diff
@@ -0,0 +1,76 @@
+package collection
+
+import (
+	"strings"
+)
+
+type Move map[string]interface{}
```

Comment:
it's not always strings ... and it will require edits to
```
func CreateModifier(modifierType string, modifierData interface{}) (ModifierInterface, error) {
...
if rangedModifier, ok := modifierData.(map[string]interface{}); ok {
```